README.md

```diff
@@ -53,12 +53,29 @@ See [MODEL ZOO](MODEL_ZOO.md).
 - pyyaml >= 5.3
 - openmpi 4.0.3 (for distributed mode)
 
-To install the dependency, please run
+### Case A:
```

Comment:
## Installation

### Case A

Install mindyolo as a package, please run:

```shell
pip install mindyolo --trusted-host ms-release.obs.cn-north-4.myhuaweicloud.com -i https://pypi.tuna.tsinghua.edu.cn/simple
```

or

```shell
pip install git+https://github.com/mindspore-lab/mindyolo.git
```

### Case B

Install mindyolo by Source Code：

```shell
git clone https://github.com/mindspore-lab/mindyolo.git
cd mindyolo
pip install -r requirements.txt
```

It is suggested to build fast coco api extension for better performance in evaluating coco data. Note that this behavior is optional:

```shell
cd $project_root/mindyolo/csrc
sh build.sh
```